client/src/App.js

```diff
@@ -1,13 +1,14 @@
-import React, { useState, useEffect } from 'react';
-import { Route } from 'react-router-dom';
-import axios from 'axios';
-import LandingPage from './containers/LandingPage.js';
+import React, { useState, useEffect } from "react";
+import { Route } from "react-router-dom";
+import axios from "axios";
+import LandingPage from "./containers/LandingPage.js";
+import { RefreshrView } from "./components";
 
 export default function App() {
   const [people, setPeople] = useState([]);
 
   const fetchPeople = async () => {
-    const response = await axios('https://swapi.co/api/people/');
+    const response = await axios("https://swapi.co/api/people/");
     setPeople(response.data.results);
```

Comment:
I think you can select for double quotes in Prettier, or just make sure the rest of the team all agrees on using double quotes for consistency.